sw/device/silicon_creator/rom_ext/rom_ext.c

```diff
@@ -379,11 +381,16 @@ static rom_error_t rom_ext_attestation_keygen(
         (hmac_digest_t *)zero_binding_value.data,
         owner_manifest->security_version, &cdi_1_key_ids,
         &curr_attestation_pubkey, cdi_1_cert, &cdi_1_cert_size));
+    HARDENED_RETURN_IF_ERROR(flash_ctrl_info_erase(
+        &kFlashCtrlInfoPageCdi1Certificate, kFlashCtrlEraseTypePage));
     HARDENED_RETURN_IF_ERROR(flash_ctrl_info_write(
         &kFlashCtrlInfoPageCdi1Certificate,
         /*offset=*/0, cdi_1_cert_size / sizeof(uint32_t), cdi_1_cert));
   }
 
+  // Restrict write and erase access to the certificate pages to the owner.
```

Comment:
I think the language of this comment is a bit unclear.  The way it reads, it sounds as if only the owner is allowed write/erase access, but really, the owner is denied write/erase access.

Reply:
Ahh good point. Fixed.